### ISSUE TITLE
Accordion and Detail: When expanded the body content isn't read by screen readers

### ISSUE THREAD

**Issue by ArakTaiRoth** [GovAlta/ui-components#1868]
# Info

A screen reader will happily read the `heading` property and anything in the `headingcontent` slot. It will also read the status of the accordion, open or closed. However, when an Accordion is opened, the body content is not read at all.

## Acceptance Criteria:

1. Body content in an accordion and detail components should be read by a screen reader when the Accordion is expanded

## More Info

Current working theory is to apply `aria-expanded` to the content `div` and set it to true or false based on the open and closed state.
